ChangeLog.md

```diff
@@ -1,3 +1,23 @@
+# ooniprobe-ios 1.1.3 [2017-03-08]
+
+Bugfixes:
+
+* Added multilingual support and Spanish, French and Italian translations
```

Comment:
Sorry to be nitpicking, but this should ideally be in the Features section, not under bugfixes.